dts/bindings/misc/renesas,rx-lvd.yaml

```
@@ -0,0 +1,82 @@
+# Copyright (c) 2025 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Renesas RX LVD driver
+
+compatible: "renesas,rx-lvd"
+
+include: [base.yaml, pinctrl-device.yaml]
+
+properties:
+  reg:
+    required: true
+
+  channel:
+    type: int
+    description: RX LVD channel
+
+  lvd-action:
+    required: true
+    type: int
+    enum: [0, 1, 2, 3]
+    description: |
+      Choose the action to be taken when the LVD is detected.
+        - 0: Reset
+        - 1: Non-maskable interrupt
+        - 2: Maskable interrupt
+        - 3: No action
```

Maybe better for readability
```suggestion
  lvd-action:
    required: true
    type: int
    enum:
      - "reset"
      - "non-maskable-interrupt"
      - "maskable-interrupt"
      - "no-action"
    description: |
      Choose the action to be taken when the LVD is detected.
```

---

It's done